pkg/ebpf/tracee.go

```diff
@@ -240,6 +240,30 @@ func New(cfg Config) (*Tracee, error) {
 		}
 	}
 
+	// Clean up signatures not chosen for tracing
```

Comment:
![image](https://github.com/aquasecurity/tracee/assets/7395852/05aa562e-e8e4-49ae-9898-affe5bc652fe)

Styling adustments needed please. I believe @josedonizetti wanted to change this approach (not to load the engine from the cobra pkg already).